forest/bonsai/simulate_gps_data.py

```diff
@@ -18,13 +18,13 @@
 import requests
 from timezonefinder import TimezoneFinder
 
+from constants import ORS_API_BASE_URL, ORS_API_DELAY, OSM_OVERPASS_URL
```

Comment:
This could conflict with another package called package `constants`, so should be `from forest.constants import ...`